### ISSUE TITLE
Error in flake8 is not reported from lint-diffs

### ISSUE THREAD

**Issue by derrix060** [AtakamaLLC/lint-diffs#20]
```
$ flake8 .
There was a critical error during execution of Flake8:
Expected `per-file-ignores` to be a mapping from file exclude patterns to ignore codes.

Configured `per-file-ignores` setting:

    mfg_tests/*.py: FS002,G001 # something
$ echo $?
1
$ git diff -z origin/dev | lint-diffs --strict

=== flake8: mine=0, always=0

$ echo $?
0
$ cat .lint-diffs 
[pylint]
# Disable pylint
extensions=

[flake8]
extensions=.py
```

**Comment by earonesty** [comment]
your per-file-ignores needs to be formatted properly

https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-per-file-ignores


for example:



```
per-file-ignores
    filename:C123,A123
    glob/*:C123
    file1, file2, file3 : E1, F2
    file4 file5: E1 E2
```
